--- a/src/Iodev/Whois/Helpers/DateHelper.php
+++ b/src/Iodev/Whois/Helpers/DateHelper.php
@@ -35,6 +35,8 @@ class DateHelper
                 : "{$m[3]}-{$m[1]}-{$m[2]}T00:00:00";
         } elseif (preg_match('/^(\d{4}-\d{2}-\d{2})\s+(\d{2}:\d{2}:\d{2})\s+\(GMT([-+]\d+:\d{2})\)$/ui', $s, $m)) {
             $s = "{$m[1]}T{$m[2]}{$m[3]}";
+        } elseif (preg_match('/^(\d{4}-\d{2}-\d{2})\s+(\d{2}:\d{2}:\d{2})\s+(CLST|CLT)$/ui', $s, $m)) {
+            $s = "{$m[1]}T{$m[2]}";
         }
         return (int)strtotime($s);
     }
